packages/workflow/src/records/handler/verify.test.ts

```diff
@@ -0,0 +1,90 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * OpenCRVS is also distributed under the terms of the Civil Registration
+ * & Healthcare Disclaimer located at http://opencrvs.org/license.
+ *
+ * Copyright (C) The OpenCRVS Authors located at https://github.com/opencrvs/opencrvs-core/blob/master/AUTHORS.
+ */
+import { createServer } from '@workflow/server'
+import { readFileSync } from 'fs'
+import * as jwt from 'jsonwebtoken'
+import { rest } from 'msw'
+import { server as mswServer } from '@test/setupServer'
+import {
+  getTaskFromSavedBundle,
+  REGISTERED_RECORD,
+  Task
+} from '@opencrvs/commons/types'
+
+function checkForVerifiedExtenstion(task: Task) {
+  return task.extension.find(
+    (e) => e.url === 'http://opencrvs.org/specs/extension/regVerified'
+  )
+}
+
+function findVerificationIp(task: Task) {
+  return task.extension.find(
+    (e) => e.url === 'http://opencrvs.org/specs/extension/regVerified'
+    //@ts-ignore
+  )!.valueString
+}
+
+describe('verify record endpoint', () => {
+  let server: Awaited<ReturnType<typeof createServer>>
+
+  beforeAll(async () => {
+    server = await createServer()
+    await server.start()
+  })
+
+  afterAll(async () => {
+    await server.stop()
+  })
+
+  it('returns OK after verifying a birth declaration', async () => {
+    const token = jwt.sign(
+      { scope: ['verify'] },
+      readFileSync('./test/cert.key'),
+      {
+        algorithm: 'RS256',
+        issuer: 'opencrvs:auth-service',
+        audience: 'opencrvs:workflow-user'
+      }
+    )
+
+    // Fetches a record from search
+    mswServer.use(
+      rest.get(
+        'http://localhost:9090/records/3bd79ffd-5bd7-489f-b0d2-3c6133d36e1e',
```

Comment:
should this refer to `SEARCH_URL` or does it matter in this case?

Reply:
![Screenshot from 2024-02-07 18-48-55](https://github.com/opencrvs/opencrvs-core/assets/32668488/8f3fd2b4-f683-463a-8796-59c73bd4fa4f)
We're calling the `getRecordById` function in the handler. So we give the `REGISTERED_RECORD` as mock data when we're calling that endpoint

Reply:
Makes sense, was just wondering if the localhost:9090 could change in some environments, would it be better to refer to the SEARCH_URL. For tests I guess it doesn't matter much